### PR TITLE
Add Browse Cards option in Deck Picker's context menu

### LINKED ISSUE #8187: Add 'View cards' to deck's context menu
**Is your feature request related to a problem? Please describe.**
Current implemenation, when you need to click dedicated menu item in left pane(to see the cards you have) is not user friendly
Looks like some legacy implemenation that could be improved

**Describe the solution you'd like**
Need to add 'View cards' menu in deck's context menu
Actual layout:
Screenshot_20210314_213512

Expected layout:
Screenshot_20210314_213512-01

**Describe alternatives you've considered**
This is the UX standard, obvious and user friendly way, no alternatives

**Additional context**
none

### PR DESCRIPTION
**Pull request by ShridharGoel**
## Purpose / Description
Add Browse Cards option in Deck Picker's context menu.

## Fixes
Fixes #8187 

## Approach
Added the option and its functionality to the menu.

## How Has This Been Tested?
Verified.

<details>
<summary>Screenshot</summary>

DeckPickerContextMenu
</details>

## Checklist
- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [ ] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Review by david-allison-1 — APPROVED** [review]
LGTM, thanks!

File: AnkiDroid/src/main/java/com/ichi2/anki/dialogs/DeckPickerContextMenu.java [hunk 1]
```diff
@@ -122,7 +130,8 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
         Collection col = CollectionHelper.getInstance().getCol(getContext());
         long did = getArguments().getLong("did");
         boolean dyn = col.getDecks().isDyn(did);
-        ArrayList<Integer> itemIds = new ArrayList<>(9);
+        ArrayList<Integer> itemIds = new ArrayList<>();
```
Comment (mikehardy) [inline AnkiDroid/src/main/java/com/ichi2/anki/dialogs/DeckPickerContextMenu.java:133]:
```suggestion
        ArrayList<Integer> itemIds = new ArrayList<>(10); // init with our fixed list size for performance
```

@Arthur-Milchior went to quite a bit of trouble to pre-size all of the arrays in the system to gain a bit of performance, keeping the initializer here and just increasing it one is optimal I believe

**Review by mikehardy — CHANGES_REQUESTED** [review]
A really minor comment, otherwise ready to go, thanks @ShridharGoel

File: AnkiDroid/src/main/java/com/ichi2/anki/dialogs/DeckPickerContextMenu.java (hunk 1 above)
Comment (ShridharGoel) [reply AnkiDroid/src/main/java/com/ichi2/anki/dialogs/DeckPickerContextMenu.java]: This has been updated.

**Review by mikehardy — APPROVED** [review]
Really sorry for the delay! LGTM

**Pull request merged**